package.json

```diff
@@ -17,6 +17,7 @@
     "ios:device:bounce": "react-native run-ios --scheme 'Bounce' --configuration Release --device",
     "ios:dev": "react-native run-ios --scheme 'Dev'",
     "ios:device:dev": "react-native run-ios --scheme 'Dev' --device",
+    "ios:device-refresh": "cd ios && fastlane match development -a co.audius.audiusmusic,co.audius.audiusmusic.bounce",
```

Comment:
nit, should we `cd -` or `cd ..` to go back to the same directory after this runs?